cabal.project

```diff
@@ -94,9 +94,9 @@ source-repository-package
 
 source-repository-package
     type: git
-    location: https://github.com/kadena-io/wai-middleware-validation.git
-    tag: f783ba7fc52e161b245d82acadffc4517bc8cdf5
-    --sha256: 182yffj2rbv2asmq589r16mb45q9d0gvdhsxvsnmjyw89dcpyadn
+    location: https://github.com/hackage-package-forks/wai-middleware-validation.git
+    tag: 34494a8ce82b6b2ed7621a6d2d8e9767f4d1bc11
+    --sha256: 1az7jcggcj275djnfsvhdg3n7hjrj6vp8rj137fxrg4hazh0hyzv
```

Comment:
Think we don't need this anymore

Reply:
Yes, can be updated to latest master of https://github.com/kadena-io/wai-middleware-validation.git